sources/packages/backend/apps/workers/src/services/cra-income-verification/cra-income-verification.service.ts

```diff
@@ -46,6 +52,40 @@ export class CRAIncomeVerificationService extends RecordDataModelService<CRAInco
     return this.repo.insert(newVerification);
   }
 
+  /**
+   * This method allows the simulation of a complete cycle of the CRA send/response
+   * process that allows the workflow to proceed without the need of the actual
+   * CRA verification happens. This is enabled based in a environment variable,
+   * that is by default disabled and should be enabled only for DEV purposes on
+   * local developer machine or on an environment where the CRA process is not enabled.
+   * !This code should not be executed on production.
+   * @param verificationId CRA verification id waiting to be processed.
+   * @returns update result.
```

Comment:
promise is not retuning

Reply:
It does not need to, right?

Reply:
ya...so the comment is also not needed right

Reply:
Now I got it 😄

Reply:
Comment removed.